model/config.go

```diff
@@ -56,8 +56,10 @@ var (
 )
 
 type ChangeDetectorConfig struct {
-	URI   string `bson:"uri" json:"uri" yaml:"uri"`
-	Token string `bson:"token" json:"token" yaml:"token"`
+	ImplementationName string `bson:"implementation_name" json:"implementation_name" yaml:"implementation_name"`
+	URI                string `bson:"uri" json:"uri" yaml:"uri"`
+	Token              string `bson:"token" json:"token" yaml:"token"`
+	IsEnabled          bool   `bson:"is_enabled" json:"is_enabled" yaml:"is_enabled"`
```

Comment:
I would put the is_enabled flag in the service flag area of the code (see earlier comment) rather than here.